src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -0,0 +1,295 @@
+﻿<UserControl x:Class="Dynamo.Wpf.Controls.InstalledPackagesControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:ui="clr-namespace:Dynamo.UI"
+             xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
+             xmlns:viewmodels="clr-namespace:Dynamo.ViewModels" 
+             d:DataContext="{d:DesignInstance Type=viewmodels:InstalledPackagesViewModel}"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <ScrollViewer  
+            VerticalScrollBarVisibility="Auto" 
+            Background="#535353" 
```

Comment:
this is a color in the dictionary, please use it.
https://github.com/DynamoDS/Dynamo/blob/bdd63b69c365d5ce5b07ff4ef20649983d49e074/src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml#L2270